ModelCatalogueCorePluginTestApp/application.properties

```diff
@@ -3,4 +3,4 @@
 app.grails.version=2.5.6
 app.name=ModelCatalogueCorePluginTestApp
 app.servlet.version=3.0
-app.version=2.2.5
+app.version=T.2.3.0.6
```

Comment:
Why is this not 2.3.1? What is the T

Reply:
T - for test.
2.3.0 Major version
x.x.x.6 immediate release for review testing (means a bug fix to an existing ongoing test)
At present we haven't released 2.3.0 (officially that is)